### PR TITLE
media-libs{ilmbase,openexr} dev-python/pyilmbase: bump to 2.5.4

### PR DESCRIPTION
**Pull request by waebbl**
Bump OpenEXR related libraries to 2.5.4
Drop 2.5.3

Bug: https://bugs.gentoo.org/656680
Bug: https://bugs.gentoo.org/762862
Bug: https://bugs.gentoo.org/746794
Closes: https://bugs.gentoo.org/762901
Package-Manager: Portage-3.0.12, Repoman-3.0.2
Signed-off-by: Bernd Waibel <waebbl@gmail.com>

### CODE REVIEW

File: media-libs/ilmbase/files/ilmbase-2.5.4-0001-disable-failing-test-on-x86_32.patch [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+diff --git a/CMakeLists.txt b/CMakeLists.txt
+index 759f359..6fab418 100644
+--- a/CMakeLists.txt
++++ b/CMakeLists.txt
+@@ -56,5 +56,5 @@ if(BUILD_TESTING)
+ 
+   add_subdirectory( HalfTest )
+   add_subdirectory( IexTest )
+-  add_subdirectory( ImathTest )
++#  add_subdirectory( ImathTest )
```
Comment (thesamesam) [inline media-libs/ilmbase/files/ilmbase-2.5.4-0001-disable-failing-test-on-x86_32.patch:20]: No point in commenting, the patch tells us what was removed

File: media-libs/ilmbase/ilmbase-2.5.4.ebuild [hunk 2]
```diff
@@ -29,17 +38,17 @@ multilib_src_configure() {
 	)
 
 	# Disable use of ucontext.h wrt #482890
-	if use hppa || use ppc || use ppc64; then
-		mycmakeargs+=(
-			-DILMBASE_HAVE_UCONTEXT_H=OFF
-		)
-	fi
+#	if use hppa || use ppc || use ppc64; then
```
Comment (thesamesam) [inline media-libs/ilmbase/ilmbase-2.5.4.ebuild:41]: Hey, what're all these comments about?

File: media-libs/openexr/openexr-2.5.4.ebuild [hunk 3]
```diff
@@ -33,9 +33,10 @@ src_prepare() {
 	# Fix path for testsuite
 	sed -i -e "s:/var/tmp/:${T}:" "${S}"/IlmImfTest/tmpDir.h || die "failed to set temp path for tests"
 
-	if use abi_x86_32 && use test; then
-		eapply "${FILESDIR}/${PN}-2.5.2-0001-IlmImfTest-main.cpp-disable-tests.patch"
-	fi
+# FIXME
```
Comment (thesamesam) [inline media-libs/openexr/openexr-2.5.4.ebuild:36]: ?

**Review by thesamesam — CHANGES_REQUESTED** [review]

File: media-libs/ilmbase/files/ilmbase-2.5.4-0001-disable-failing-test-on-x86_32.patch (hunk 1 above)
Comment (waebbl) [reply media-libs/ilmbase/files/ilmbase-2.5.4-0001-disable-failing-test-on-x86_32.patch:20]: Guess I commented it at first to test whether this works. I'm removing the line.

File: media-libs/ilmbase/ilmbase-2.5.4.ebuild (hunk 2 above)
Comment (waebbl) [reply media-libs/ilmbase/ilmbase-2.5.4.ebuild:41]: I have neither of these machines available to test whether these are still needed. Should have been fixed according to the cmake file, but I couldn't test it.

**Pull request closed without merging**